website/docs/components/button/partials/guidelines/guidelines.md

```diff
@@ -121,13 +116,42 @@ Use `external-link` when using the Button for external links. In most cases, con
 
 <Hds::Button @color="secondary" @text="Authenticate with GitHub" @icon="external-link" @iconPosition="trailing" @href="https://hashicorp.com" />
 
-## Button set
+## Links in Buttons
+
+!!! Insight
+
+While we support using links within buttons, please strive for the correct semantic use. Learn more about [how semantic use helps make our products more conformant](/components/button?tab=accessibility#button-vs-link).
```

Comment:
```suggestion
While we support using links as buttons (that is, with the visual appearance of a button), please strive for the correct semantic use. Learn more about [how semantic use helps make our products more conformant](/components/button?tab=accessibility#button-vs-link).
```

Reply:
I ended up taking this out since it was essentially sending the same message as the content below but in different words.